integration_tests/__tests__/list_tests.test.js

```diff
@@ -12,10 +12,17 @@
 const runJest = require('../runJest');
 
 describe('--listTests flag', () => {
-  it('causes tests to be printed out as JSON', () => {
+  it('causes tests to be printed in different lines', () => {
     const {status, stdout} = runJest('list_tests', ['--listTests']);
 
     expect(status).toBe(0);
     expect(() => JSON.parse(stdout)).not.toThrow();
```

Comment:
Yeah, I just added something similar but by snapshotting the output.. hope that is fine